sbi/utils/mcmc_utils.py

```diff
@@ -0,0 +1,82 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Apache License Version 2.0, see <https://www.apache.org/licenses/>
+
+import logging
+from typing import Callable, Optional, Union
+
+from numpy import ndarray
+from torch import Tensor
+
+from sbi.samplers.mcmc import SliceSamplerSerial, SliceSamplerVectorized
+from sbi.utils.torchutils import tensor2numpy
+
+
+def get_mcmc_samples(
+    potential_fn: Callable,
+    initial_params: Union[ndarray, Tensor],
+    num_samples: int = 1000,
+    vectorized: bool = True,
+    num_workers: int = 1,
+    num_chains: int = 1,
```

Comment:
I think the default value here should match the default value for `MCMCPosterior` (I think it's 20)